src/test/CxPlatTests.h

```diff
@@ -135,10 +136,13 @@ static const GUID CXPLAT_TEST_DEVICE_INSTANCE =
 #define IOCTL_CXPLAT_RUN_THREAD_BASIC \
     CXPLAT_CTL_CODE(6, METHOD_BUFFERED, FILE_WRITE_DATA)
 
-#define IOCTL_CXPLAT_RUN_THREAD_WAIT_TIMEOUT \
+#define IOCTL_CXPLAT_RUN_THREAD_ASYNC \
```

Comment:
It'd make the diff nicer to make `IOCTL_CXPLAT_RUN_THREAD_ASYNC ` number 9 (last) instead.